test/atomic-load-store.ll

```diff
@@ -18,18 +17,18 @@ entry:
 ; CHECK: Variable [[#]] [[#PTR:]]
   %0 = alloca i32
 
-; CHECK: AtomicStore [[#PTR]] [[#DeviceScope]] [[#Relaxed]] [[#]]
+; CHECK: AtomicStore [[#PTR]] [[#CrossDeviceScope]] {{.+}} [[#]]
```

Comment:
To double check, the memory order is still Relaxed, you just don't want to reuse [[#CrossDeviceScope:]] constant in the check, right?

Reply:
Correct, re-using the same "named" variable in lit is not quite possible (or I've not figured out how), and instead of just picking something like `Const0` it seemed more desirable to pattern match one and keep the other fully described (it retains most of the information about what is being tested).